HOMEWORK_9/EX2.CPP

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+
+void toUppercase(char str[])
+{
+    for (int i = 0; str[i] != '\0'; ++i) {
+        if (str[i] >= 'a' && str[i] <= 'z') {
+            str[i] = str[i] - 'a' + 'A';
```

Comment:
move this 'a' + 'A' into variable outside of the loop to not repeat this calculation every iteration of the loop

Reply:
done